cbw_api_toolbox/cbw_api.py

```diff
@@ -461,3 +463,21 @@ def delete_security_issue(self, security_issue_id):
             return None
 
         return CBWParser().parse_response(CBWSecurityIssue, response)
+
+    def fetch_importer_scripts(self, params=None):
```

Comment:
Would it be a good idea to have a param to disable attachement download ? It would be usefull when you want to scan modern windows or linux, and prevent cab file download.